### PR TITLE
Removed extra margin from help-icon

### PR DESCRIPTION
**Pull request by siddhant1**

### CODE REVIEW

File: src/components/header/renderer.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ class Header extends React.PureComponent<Props, State> {
 
     const HelpButton = (
       <div className="header-button help" onClick={() => this.setState(current => ({ ...current }))}>
```
Comment (domoritz) [inline src/components/header/renderer.tsx]: We already have the class `help` here. No need for another class.

Comment (siddhant1) [reply src/components/header/renderer.tsx]: fixed!

**Comment by siddhant1** [comment]
Should I merge?

**Comment by domoritz** [comment]
Yes

**Pull request merged**